### ISSUE TITLE
Spectral rendering gives heavy blue tint in renders

### ISSUE THREAD

**Issue by SergeyShlyaev** [mmp/pbrt-v3#276]
Spectral rendering produces images with heavy blue tint for some reason.

To enable spectral rendering, I've recompiled pbrt with PBRT_SAMPLED_SPECTRUM defined in core/pbrt.h 

Using Mac Os Catalina with Xcode 11.3.1
![spec_01](https://user-images.githubusercontent.com/63272131/79079432-1e86f880-7cdd-11ea-8e77-0c9f9acf63c9.jpg)

![spec_02](https://user-images.githubusercontent.com/63272131/79079443-25ae0680-7cdd-11ea-9478-4ebc01b49838.jpg)

**Comment by mmp** [comment]
This is basically due to a bad design decision (that will be fixed in pbrt-v4). Most lights in pbrt-v3 take multiple parameters to specify their emission: e.g. the diffuse area light has both "scale" and "L".  In turn, when its emission is calculated, the standard illuminant is included twice (multiplied with itself), which in turn gives that blue hint.

As a workaround, you could remove those "scale" parameters from the lights you care about.

**Comment by SergeyShlyaev** [comment]
Yeah, I tried multiplying light scales by D65 white, which is RGB = (1.20, 0.95, 0.91).
It helped with the dragon scene.

But for living room scene it still produce heavy blue tint under couch pillows as on my second image. Any suggestions?

**Comment by mmp** [comment]
Hm, I'm not sure about that blue by the pillows. My best advice would be to render just one of those pixels and add debugging print statements that describe the spectral calculations being done, and then see where they start to go not-so-white...
